runbotics/runbotics-desktop/src/utils/credentialAttributesMapper.ts

```diff
@@ -0,0 +1,25 @@
+import { DecryptedCredential } from 'runbotics-common';
+
```

Comment:
could use little documentation for future generations. in this convention
 
![image](https://github.com/user-attachments/assets/4dfc35ed-dc37-457a-bee2-8326c78a42ea)